src/dft.jl

```diff
@@ -1,123 +1,133 @@
+using ITensorMPS: apply, MPO
```

Comment:
```suggestion
using ITensorMPS: MPO, apply
```
It's conventional to list types first followed by functions.